fe/projects/AspNetCore/src/areas/Products/components/ProductDetail/ProductDetail.js

```diff
@@ -112,123 +117,183 @@ function ProductDetail(props) {
-                    <h2 className="product-detail__brand subtitle is-6">{props.byLabel} <a href={props.brandUrl}>{props.brandName}</a></h2>
-                    {props.outletTitle &&
-                        <div className="product-details__discount">{props.outletTitleLabel} {props.outletTitle}</div>     
-                    }
-                    {props.inStock && props.availableQuantity && props.availableQuantity > 0 &&
-                        <div className="product-detail__in-stock">
-                            {props.inStockLabel} {props.availableQuantity}
-                            {props.expectedDelivery && 
-                                <div className="product-detail__expected-delivery">{props.expectedDeliveryLabel} {moment.utc(props.expectedDelivery).local().format("L")}</div>
-                            }
-                        </div>
-                    }
-                    {props.inOutlet && props.availableOutletQuantity && props.availableOutletQuantity > 0 &&
-                        <div className="product-detail__in-stock">
-                            {props.inOutletLabel} {props.availableOutletQuantity}
-                        </div>
-                    }
-                    {props.isAuthenticated && 
-                        <div className="product-detail__add-to-cart-button">
-                            {props.isProductVariant ? (
-                                <div className="row">
-                                    <Button type="text" variant="contained" color="primary" onClick={() => setIsModalOpen(true)}>
-                                        {props.basketLabel}
-                                    </Button>
-                                </div>
-                            ) : (
-                                <div className="product-detail__add-to-cart-button">
-                                    <Button type="text" variant="contained" color="primary" onClick={() => setIsSidebarOpen(true)}>
-                                        {props.basketLabel}
-                                    </Button>
+            <div className="product-detail__container">
+                <div className="product-detail__head columns is-desktop">
+                    <div className="product-detail__gallery-column">
+                        <div className="column product-detail__desktop-gallery">
+                            <div className="is-flex is-flex-wrap product-detail__product-gallery">
+                                {props.images && props.images.length > 0 && images.map((image, index) => {
+                                    return (
+                                        <LazyLoad offset={LazyLoadConstants.defaultOffset()} className="product-detail__desktop-gallery__desktop-product-image" key={index}>
+                                            <ResponsiveImage sources={image.sources} imageSrc={image.src} imageAlt={image.alt} />
```

Comment:
**suggestion:** paramete imageSrc is wrong passed. Should be image.imageSrc instead of image.src